src/game/client/display.cpp

```diff
@@ -74,9 +78,110 @@ void Display::Reset()
 {
-#ifdef GAME_DLL
-    Call_Method<void, Display>(PICK_ADDRESS(0x00421870, 0x007BDB67), this);
-#endif
+    /*
+    if (Is_Movie_Playing() && m_videoStream->Is_Frame_Ready()) {
+        m_videoStream->Decompress_Frame();
+        m_videoStream->Render_Frame(m_videoBuffer);
+
+        // get next frame
+        if (!m_videoStream->Is_Done()) {
+            m_videoStream->Next_Frame();
+        } else if (m_someLogoMovieInt2 >= 0 || m_someLogoMovieInt1 >= 0) {
+            // some sort of copyright text timing, doesn't ever show in retail......
+            if (!m_unkInt) {
+                if (m_unkDisplayString != nullptr) {
+                    delete m_unkDisplayString;
+                }
+
+                m_unkDisplayString = g_theDisplayStringManager->New_Display_String();
+                m_unkDisplayString->Set_Text(g_theGameText->Fetch("GUI:EACopyright"));
+
+                Utf8String font_name;
+                int point_size = 0;
+                bool bold = false;
+
+                if (g_theGlobalLanguage && g_theGlobalLanguage->Copyright_Font().Name().Is_Not_Empty()) {
+                    FontDesc cpy_fnt = g_theGlobalLanguage->Copyright_Font();
+                    font_name = cpy_fnt.Name();
+                    point_size = cpy_fnt.Point_Size();
+                    bold = cpy_fnt.Bold();
+                } else {
+                    font_name = "Courier";
+                    point_size = 12;
+                    bold = true;
+                }
+
+                int size = g_theGlobalLanguage->Adjust_Font_Size(point_size);
+                GameFont *font = g_theFontLibrary->Get_Font(font_name, size, bold);
+                m_unkDisplayString->Set_Font(font);
+
+                m_unkInt = rts::Get_Time();
+            }
+
+            if (m_someLogoMovieTime + m_someLogoMovieInt1 < (unsigned int)rts::Get_Time()) {
+                if (m_unkInt + m_someLogoMovieInt2 < (unsigned int)rts::Get_Time()) {
+                    m_someLogoMovieInt1 = -1;
+                    m_someLogoMovieTime = 0;
+                    m_unkInt = 0;
+                    m_someLogoMovieInt2 = -1;
+                }
+            }
+        } else {
+            Stop_Movie();
+        }
+    }
+    */
+
+    if (Is_Movie_Playing() && m_videoStream->Is_Frame_Ready()) {
+        m_videoStream->Decompress_Frame();
+        m_videoStream->Render_Frame(m_videoBuffer);
+
+        if (m_videoStream->Frame_Index() != m_videoStream->Frame_Count() - 1) {
+            m_videoStream->Next_Frame();
+        } else if (m_someLogoMovieInt2 < 0 && m_someLogoMovieInt1 < 0) {
+            Stop_Movie();
+        } else {
+            // some sort of text timing, we never see the text cause its color is black
+            if (!m_unkInt) {
+                if (m_unkDisplayString != nullptr) {
+                    Delete_Instance(m_unkDisplayString);
+                }
+
+                m_unkDisplayString = g_theDisplayStringManager->New_Display_String();
```

Comment:
Based on this the unknown string is something like `m_copyrightString` or some such?

Reply:
To my best ability to tell this is a general purpose slot used in sort of emergency cases.
W3DDisplay::Draw for instance, seems would use it for some debug prints, not clear what.